src/library/trackprocessing.h

```diff
@@ -64,7 +65,7 @@ class ModalTrackBatchProcessor
 
     const Duration m_minimumProgressDuration;
 
-    bool m_bAborted;
+    std::atomic<bool> m_aborted;
```

Comment:
A slot is always supposed to be invoked from the event loop thread if not stated otherwise. Please explain why this atomic is needed. I don't think it is. Everything happens on the UI thread.

Reply:
Oh Yes, I was confused because of the while call above.